packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -205,6 +207,7 @@ describe('<DebugFailedTest/>', () => {
       hasStdout: false,
       hasScreenshots: false,
       hasVideo: false,
+      hasReplay: false,
```

Comment:
Add a test case for the new button below.

Reply:
Updated.